sled-agent/src/services.rs

```diff
@@ -507,6 +507,21 @@ impl ServiceManager {
                             err,
                         })?;
                 }
+                ServiceType::Dendrite { asic } => {
+                    info!(self.log, "Setting up dendrite service");
+                    running_zone
+                        .run_cmd(&[
+                            crate::illumos::zone::SVCCFG,
+                            "-s",
+                            &smf_name,
+                            "setprop",
+                            &format!("config/asic={}", asic),
```

Comment:
Yes.  We need the `svccfg refresh`.

My other issue was caused because the `dendrite` service description was enabling the service by default, so it was already running by the time we updated the property.

I'll update the PR with both the refresh and a new `dendrite` commit/sha256.

Thanks for calling this out.